content/2_Bronze/Intro_Sets.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 id: intro-sets
-title: (Optional) Introduction to Sets & Maps
+title: Introduction to Sets & Maps
```

Comment:
we should not recommend skipping this module - a lot of internal solutions use these and we expect users to be familiar with sets and maps

(even if for many Bronze problems the constraints are low enough so that they are not required to pass the time limit)